src/ui/public/vis/editors/default/default.js

```diff
@@ -47,15 +47,14 @@ const defaultEditor = function ($rootScope, $compile) {
       }
     }
 
-    render(visData, searchSource, updateStatus, uiState) {
+    render(savedObj, params) {
       let $scope;
+      const { uiState, timeRange, appState } = params;
```

Comment:
:art: Destructure directly in the `render(savedObj, { uiState, timeRange, appState })`